.github/workflows/sync-from-external-sources.yml

```diff
@@ -70,4 +70,27 @@ jobs:
           body: |
             - Update SAS Strings from matrix-doc.
           branch: sync-sas-strings
+          base: develop
+
+  sync-analytics-plan:
+    runs-on: ubuntu-latest
+    # Skip in forks
+    if: github.repository == 'vector-im/element-android'
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run analytics import script
+        run: ./tools/import_analytic_plan.sh
+      - name: Create Pull Request for analytics plan
+        uses: peter-evans/create-pull-request@v3
```

Comment:
another possibility is to `.jar` the generated output as part of the schema repository and upload to github packages, that way dependabot can handle updates and we can ensure the generated artifact compiles in isolation

Reply:
Yes, @pixlwave told me about that possibility. It's probably a cleaner solution, but I have never done that before 🙈